.github/workflows/plugin-update.yml

```diff
@@ -72,8 +79,9 @@ jobs:
             gh pr create \
             --head "$VAULT_BRANCH" \
             --reviewer "$REVIEWER" \
+            --assignee "$REVIEWER" \
             --title "Update ${{ inputs.plugin }} to v${{ inputs.version }}" \
-            --body "This PR was generated by a GitHub Action. Full log: https://github.com/hashicorp/vault/actions/runs/${{ github.run_id }}"
+            --body "This PR was generated by a GitHub Action. Full log: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
```

Comment:
Do we want a check on `github.repository == 'hashicorp/vault*` here?

Reply:
No, this action will be called by either vault or vault-enterprise and the `github.repository` value will be the correct by definition. No conditionals necessary

Reply:
I'm wondering about invoking this with a non-Hashi repo.

Reply:
Ah yeah, I see.